src/containers/AccountConnection.jsx

```diff
@@ -120,15 +121,15 @@ class AccountConnection extends Component {
       })
   }
 
-  updateAccount (connector, account, values) {
+  updateAccount (connector, account, values, successTimeout) {
     Object.assign(account.auth, values)
 
     this.setState({ submitting: true })
 
     return this.store.updateAccount(connector, account, values)
     .then(account => {
       this.setState({ account: account })
-      return this.store.runAccount(connector, account)
+      return this.store.runAccount(connector, account, successTimeout)
```

Comment:
`SuccessTimeout`is defaulted to 20 seconds in https://github.com/cozy/cozy-collect/blob/master/src/lib/konnectors.js#L128 if undefined.